### DISCUSSION TITLE
Dear Professor Sired

### DISCUSSION THREAD

**Discussion by FDDZG** [openearth/aeolis-python#230] [Q&A]
Dear Professor Sierd, my name is Zhu Jialin from Lanzhou University of Technology. I am a master majoring in wind-blown sand fluid mechanics. Recently I have been using your Aeolis program to calculate some content. Now I have encountered some problems and would like to consult you. I configured a two-dimensional environment in a calculation example, but there was no result in the final calculation result of bed change zb. Here, I sincerely ask you what caused this problem. The configuration file is in the compressed package file. Sorry to bother you. Finally, I wish you a smooth work and a happy life.
[file.zip](https://github.com/user-attachments/files/18104978/file.zip)

**Comment by Sierd** [comment]
Dear @FDDZG ,

I tried to run your model but the input seem incomplete. Could you send me your complete set of input files (including the file structure, I notice you must use a 'data' folder yourself that you did not include in the ZIP file. If I can reproduce your model I can help you better.
